### ISSUE TITLE
Dark mode ?

### ISSUE THREAD

**Issue by Cipher66hz** [CE-Programming/CEmu#490]
---------------------------------------------
**Operating System:**Linux Mint 21.3 x86_64
**CEmu version:** … Precompiled CEmu_nightly_linux64_qt6_static
**Describe your issue:**  
When building from source in QT-Creator the application launched in a dark-mode type setting, however in the binary release this isn't applicable. Im not familiar with QT-Creator, so I wasnt sure how to compile from source and make one individual executable file.  (I also wasnt able to find information online about doing this).
Please pardon me being a bit of a newbie with regards to QT-Application development lol
… (including, if applicable what you expect vs. what happens)  

TLDR: How can I enable a dark mode in the binary executable. Having issues compiling from source and having it as its own standalone program (like the CEmu_nightly_qt6_static version).

What are the steps to reproduce this issue?
-------------------------------------------
1. Download static_nightly release from https://github.com/CE-Programming/CEmu/releases/tag/nightly
2. Run it, no dark mode? 

Any logs, error output, screenshot, other comments...?
------------------------------------------------------
N/A

Thank you for your patience :)

**Comment by adriweb** [comment]
There's nothing to change on the code side, this is rather a platform / qt theming issue.
You may try launching the binary with these env vars to see if it's better, but other than that, I'm not sure...

`QT_STYLE_OVERRIDE=Adwaita-Dark`
`QT_STYLE_OVERRIDE=Kvantum` (see [install readme here](https://github.com/tsujan/Kvantum/blob/master/Kvantum/INSTALL.md))

As for static binaries, this requires more effort (you first have to get (build, likely) your own static Qt SDK. but the way we set things up with cmake+vcpkg, it is pretty much automatic when you choose the appropriate preset. This will be explained in more details once we write the doc...

**Comment by Cipher66hz** [comment]
Hmm. I already have Kvantum & QT5 Config settings configured. I put `QT_STYLE_OVERRIDE=Adwaita-Dark` in my .desktop file for launching CEmu, but that made no difference.

Its no big deal though! Just maybe something to consider adding in future, user input-able theming / color schemes. 
I'll rock it white for now :D 

Thanks again for your help !

**Comment by adriweb** [comment]
I don't know if that works. From the terminal what happens when you launch the binary with the env var exported before though?

**Comment by Cipher66hz** [comment]
![image](https://github.com/user-attachments/assets/6fc438c1-e89b-4b1f-9737-042557e7998c)

Yeah tried using both (just for redundancy), still white :/

![image](https://github.com/user-attachments/assets/bb5bad61-2cc8-4a66-8678-6b9c6e20da9a)

As I said, no biggie , so if you've got no ideas s'all good

**Comment by Cipher66hz** [comment]
![image](https://github.com/user-attachments/assets/a9a2b674-b819-4f61-b984-c0f870179ca5)

Update, realized I had to pass it through on the same line.

Only Windows and Fusion are the available styles , despite the fact that I have quite a few QT themes installed on my system. It must be referencing another directory to source the themes.

Again, thank you for your patience.
